Department.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
```

Comment:
added strict types declaration:

```suggestion
declare(strict_types=1);

```